lib/pdk/cli/util/command_redirector.rb

```diff
@@ -0,0 +1,31 @@
+require 'tty-prompt'
+
+module PDK
+  module CLI
+    module Util
+      class CommandRedirector < TTY::Prompt::AnswersCollector
+        def pastel
+          @pastel ||= Pastel.new
+        end
+
+        def target_command(cmd)
+          @command = cmd
+        end
+
+        def run
+          @prompt.print _('Did you mean \'')
+          @prompt.print pastel.bold(@command)
+          @prompt.puts _('\'?')
```

Comment:
Translators will flag this as untranslatable.

It does work as `@prompt.print _('Did you mean \'%{command}\'?') % { command: pastel.bold(@command) }`.